dotnet/src/Connectors/Connectors.Memory.Kusto/README.md

```diff
@@ -0,0 +1,35 @@
+﻿# Microsoft.SemanticKernel.Connectors.Memory.Kuato
+
+This connector uses Kusto (Azure Data Explorer)[https://learn.microsoft.com/en-us/azure/data-explorer/] to implement Semantic Memory.
+
+## Quick start
+
+1. Create a cluster and database in Kusto (Azure Data Explorer) - see https://learn.microsoft.com/en-us/azure/data-explorer/create-cluster-and-database?tabs=free
+
+2. To use Kusto as a semantic memory store:
+
+```csharp
+var connectionString = new KustoConnectionStringBuilder("https://mycluster.kusto.windows.net").WithAadUserPromptAuthentication();
+KustoMemoryStore memoryStore = new(connectionString, "MyDatabase");
+
+IKernel kernel = Kernel.Builder
+    .WithLogger(ConsoleLogger.Log)
+    .WithOpenAITextCompletionService(modelId: TestConfiguration.OpenAI.ModelId, apiKey: TestConfiguration.OpenAI.ApiKey)
+    .WithOpenAITextEmbeddingGenerationService(modelId: TestConfiguration.OpenAI.EmbeddingModelId,apiKey: TestConfiguration.OpenAI.ApiKey)
+    .WithMemoryStorage(memoryStore)
+    .Build();
+```
+
+## Important notes
+
+### Coisine similarity
+At this time, coisine similiarity is not baked in into Kusto. It is add to the database hosting the collections when the first collection is created.  
+This is done by creating a function in the database. 
+This function is not deleted when the last collection is deleted. This is done to avoid the cost of creating the function each time a collection is created. 
+If you want to delete the function, you can do it manually using the Kusto explorer. The function is called `series_cosine_similarity_fl` and is located in the `Functions` folder of the database.
```

Comment:
That's too much info :)
Just say something like:

A function to calculate cosine similarity is automatically added to the Kusto data when the first collection is created. This function (`series_cosine_similarity_fl`) is not removed. You might want to delete it manually if you stop using the Kusto database as a semantic memory store.

Reply:
done